src/pip/_internal/cli/cmdoptions.py

```diff
@@ -305,11 +305,14 @@ def exists_action():
 
 cert = partial(
     PipOption,
-    "--cert",
-    dest="cert",
-    type="path",
-    metavar="path",
-    help="Path to alternate CA bundle.",
+    '--cert',
+    dest='cert',
+    type='path',
+    metavar='path',
+    help="Path to PEM-encoded CA certificate bundle. "
+         "If provided, overrides the default. "
+         "See 'SSL Certificate Verification' in pip documentation "
+         "for more information.",
```

Comment:
It seems the linter has some specific rules regarding indentation. Try wrapping the string in parantheses:

```python
help=(
    "Path to PEM-encoded CA certificate bundle. "
    "If provided, overrides the default. "
    "See 'SSL Certificate Verification' in pip documentation "
    "for more information."
),
         "for more information.",
```